configure.ac

```diff
@@ -1455,9 +1455,12 @@ dnl Check for libnatpmp (optional).
 if test "$use_natpmp" != "no"; then
   TEMP_CPPFLAGS="$CPPFLAGS"
   CPPFLAGS="$CPPFLAGS $NATPMP_CPPFLAGS"
-  AC_CHECK_HEADERS([natpmp.h],
-                   [AC_CHECK_LIB([natpmp], [initnatpmp], [NATPMP_LIBS="$NATPMP_LIBS -lnatpmp"], [have_natpmp=no], [$NATPMP_LIBS])],
-                   [have_natpmp=no])
+  AC_CHECK_HEADERS([natpmp.h], [], [have_natpmp=no])
+
+  if test "$have_natpmp" != "no"; then
```

Comment:
I think this is one of those cases where you need the funky xsyntax.

`$have_natpmp` is either "no" or empty. IIRC empty can screw up `test` on some shells.

This is usually avoided by either initializing to something or doing: `if test "x$have_natpmp" != "xno"; then`.

Reply:
>  IIRC empty can screw up test on some shells.

I think bash fixed that in 1996, and we have since removed all the x-prefix usage from our build system. It'd be good to not have to reintroduce it.

Reply:
Lol, you trying to tell me something? :)